julearn/model_selection/tests/test_available_searchers.py

```diff
@@ -0,0 +1,35 @@
+import pytest
```

Comment:
Missing
```Python
# Authors: Sami Hamdan <s.hamdan@fz-juelich.de>
# License: AGPL
```

Reply:
Added both of us: as we did for all the other files in julearn (except for the examples)